apps/system/js/value_selector/spin_date_picker.js

```diff
@@ -363,6 +369,10 @@ var SpinDatePicker = (function() {
       }).bind(this));
     },
 
+    getDateValue: function() {
+      return this._value.toISOString().split('T')[0];
```

Comment:
ISO string is always `YYYY-MM-DD` until year 10000, so this can be `.substr(0, 10)`.